### PR TITLE
Small cosmetic fixes for UserRating and Contests views

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- added cosmetic enhancements to UserRating
- changed Contests view: added featured contests

https://user-images.githubusercontent.com/58667063/196805726-ff5b47b1-58fe-4bdd-b251-b4116f1aa837.mov

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+        style = jso {
+            width = 24.rem
+        }
+    }
+}
+
+private fun ChildrenBuilder.stayTuned() {
+    div {
+        className = ClassName("card-body d-flex flex-column align-items-start")
+        ReactHTML.strong {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt:81]:
```suggestion
        strong {
```
and so on in this file

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ProposeYourContest.kt [hunk 2]
```diff
@@ -19,57 +19,44 @@ import kotlinx.js.jso
...
 
-                        strong {
-                            className = ClassName("d-inline-block mb-2 text-success")
-                            +"saveourtool@gmail.com"
-                        }
-                    }
-                }
+            strong {
+                className = ClassName("d-inline-block mb-2 text-success")
+                +"saveourtool@gmail.com"
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ProposeYourContest.kt:59]: maybe make it clickable?
Like

```suggestion
            a {
                href = "mailto:saveourtool@gmail.com"
                +"saveourtool@gmail.com"
```

**Review by sanyavertolet — APPROVED** [review]
I love how it looks

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt:81]: thx

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ProposeYourContest.kt (hunk 2 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ProposeYourContest.kt:59]: very good idea

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/FeaturedContests.kt]: fixed

**Pull request merged**